### DISCUSSION TITLE
Keyboard shortcut to move or resize window that is currently under the mouse pointer

### DISCUSSION THREAD

**Discussion by bradhs** [rxhanson/Rectangle#867] [Ideas]
Hello,

I'm looking for an app that will move or resize the window that is directly under the mouse pointer with a simple keyboard shortcut.

I use HyperDock by Christian Baumgart but I think he's stopped supporting it.

<img width="622" alt="image" src="https://user-images.githubusercontent.com/51719730/181690410-453fa6e1-5a23-4e5a-99da-87d8543b7b38.png">

**Comment by bradhs** [comment]
This seems to come close to what I am looking for as well, but it's out of date too: https://github.com/dmarcotte/easy-move-resize

**Comment by rxhanson** [comment]
This is built into Rectangle Pro (https://rectangleapp.com/pro). Note that Rectangle Pro will only resize from the bottom right corner, and this is intentional because resizing from the other corners can be really janky.

You can also check out Hummingbird (https://github.com/finestructure/Hummingbird).

I don't have plans to add this into Rectangle.
